packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -79,6 +79,7 @@ import {
   PerformedWork,
   Placement,
   Hydrating,
+  Callback,
```

Comment:
I’m not sure this is really the right flag for this.

Reply:
Yeah I'm not either but I couldn't come up with a better one. I'm not even sure it's essential to do this in the effect phase but it fell out of using this flag so that's where it is for now